src/FxKit/Option/Option.EscapeHatch.cs

```diff
@@ -69,7 +69,7 @@ public static partial class Option
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
     public static async Task<T?> ToNullableAsync<T>(this Task<Option<T>> source)
         where T : class =>
-        (await source).TryGet(out var result) ? result : null;
+        (await source.ConfigureAwait(false)).TryGet(out var result) ? result : null;
```

Comment:
Has the lack of `ConfigureAwait(false)` caused any issues? Just curious. I've never had to add them myself.

Reply:
It is just a recommendation for libraries. I recommend this article: https://blog.stephencleary.com/2023/11/configureawait-in-net-8.html